lib/balena-lint.ts

```diff
@@ -93,7 +93,12 @@ const findFiles = async (
 		paths.map(async (p) => {
 			if ((await fs.stat(p)).isDirectory()) {
 				files.push(
-					...(await globAsync(`${p}/**/*.@(${extensions.join('|')})`)),
+					...(await globAsync(`${p}/**/*.@(${extensions.join('|')})`, {
+						// Ignore unusual errors while reading directories.
+						// Avoids EPERM scandir errors in Windows.
+						// See: https://github.com/isaacs/node-glob/issues/284
+						strict: false,
```

Comment:
This could in theory mask errors that a developer might wish to be warned about (e.g. actual permission errors), however presumably such errors would also affect the functioning of the application or the CI builds/tests and should be caught by testing rather than linting, so I think it is a fair enough workaround to a `node-glob` issue. 👍